raiden/tests/scenarios/pfs7_simple_path_rewards.yaml

```diff
@@ -59,32 +59,42 @@ scenario:
             - deposit: {from: 2, to: 1, total_deposit: 1000}
             - deposit: {from: 3, to: 2, total_deposit: 1000}
 
+      - serial:
+          name: "Deposit in the other directions"
+          tasks:
+            - wait: 10
```

Comment:
why is the wait necessary here?

Reply:
it is just for security reasons. Sometimes it takes a while until the SP sees the deposit